tests/key_field_sample.fbs

```diff
@@ -0,0 +1,21 @@
+namespace keyfield.sample;
```

Comment:
can we put the key-field schema and all related generated code in their own directory?

Reply:
Sure